### DISCUSSION TITLE
Just saying Thank You!

### DISCUSSION THREAD

**Discussion by arsradu** [kyujin-cho/pixel-volte-patch#6] [General]
Hello,

Just wanted to say a huge THANK YOU for this! I've just tested this on Vodafone and Orange RO (MCC-MNC: 226-01 & 226-10) on a Pixel 6, and BOTH VoLTE and VoWIFI are working perfectly! With and without root.

This is an amazing achievement! And I just want to congratulate the dev(s) for their incredible work.

I don't have a 5G plan, and I don't live in a 5G area, so I couldn't test this. I saw it's still work in progress, or at least being discussed. But for me, the way it works right now, and especially the fact that it can work without root, is amazing!

Once again, congratulations! And thank you very much for all the effort!

**Comment by mariusbalanuca** [comment]
I also want to say thanks and I want to buy you a coffee, maybe you can add a link for that in the README.md

🙏

**Reply by kyujin-cho** [reply]
Thanks, but due to some strange regulations (somewhat related tax maybe?) at our country it is really hard to accept donations from overseas. Anyway I really appreciate it.

**Comment by BBCarabina** [comment]
I also want to say thank you, and to promise 100 bucks to the guy that makes it work on snapdragon devices too!

**Comment by rimasx** [comment]
Big Thank You from Estonia.

**Comment by templatetuners** [comment]
I can confirm, Vodafone RO works like charm on Pixel 6

**Reply by skatepark99** [reply]
Hi, im also on vodafone but it seems on my pixel 7 it doesnt work. Could you help me?

**Comment by luxdave** [comment]
Working perfectly on a Pixel 6 in Belgium (Orange BE). Thank you!

**Comment by skatepark99** [comment]
I have a Pixel 7 and im on vodafone and i followed all the steps but it doesnt work. Could somebody help me? Here is my discord Theo1#6634

**Comment by arsradu** [comment]
What is the OS version or build number?

**Reply by skatepark99** [reply]
i have the latest android update is build number is TQ2A.23050.002

**Reply by skatepark99** [reply]
build number: TQ2A.230505.002 this is the correct one

**Comment by arsradu** [comment]
Ok. So it's Android 13 no beta, right? 
What happens when you try to activate volte and vowifi with Pixel IMS app?

**Reply by skatepark99** [reply]
I have android 13, no beta. So at the final step when i check the volte and vowifi i restart the phone as the instruction says and when i get back to pixel ims its still unregistered ims. I checked the developer setting and every time i restart the phone wireless debugging is off and shizuku the same. I tested this with latest verison and older verison. Any suggestions?

**Reply by arsradu** [reply]
Is Shizuku configured correctly? At first, I also thought I should reactivate Shizuku every after each restart. But nope, no need for that.
You need to activate it, start the service and all, when you want to make changes, which should only be once when you set up everything (and again after an OS update). Should work perfectly fine after that.

Can you post a screenshot of Shizuku running and your Pixel IMS configuration screen?
This is what it looks like in my case, on Orange RO. But I've tested this before, on Vodafone RO, as well, and it was working perfectly fine. So I'm pretty sure it should work.

![Screenshot_20230511-174154](https://github.com/kyujin-cho/pixel-volte-patch/assets/86431007/b92c530b-cc48-4be8-abc6-fc7f003694b2)
![Screenshot_20230511-174307](https://github.com/kyujin-cho/pixel-volte-patch/assets/86431007/c7601161-fe21-41de-b4d4-a42c5bdef61d)

**Reply by skatepark99** [reply]
I tried again, when i do it, it looks the same as in your screenshots, but when i restart the phone, wireless debugging is off, shizuku is not running and i have to start it again and pixel ims status is unregistered. I dont know where is the problem.

**Reply by arsradu** [reply]
But did you try to make a call? It shows unregistered in the Pixel IMS app, but it works perfectly.
